### ISSUE TITLE
disable upload

### ISSUE THREAD

**Issue by amille44420** [django-ckeditor/django-ckeditor#213]
Hi,

I'm looking into using ckeditor for my rich text field but I'm not interested by the upload feature. I don't want to let anyone (even super user) upload anything.

By searching on that Github, I saw I had to remove the urls from my settings to disable uploads. And yet I'm having a critical issue (NoReverseMatch).

Is there currently any way to disable uploads? Or is that feature lost? I believe we should always have the choice to disable that feature (what if we want only rich text?).

My best regards,

**Comment by riklaunim** [comment]
Did you checked what and where caused that NoReverseMatch? If you don't want uploads you also should configure CKEditor not to display that widget in its toolbar.

**Comment by riklaunim** [comment]
In current master I'm moving all upload specific code to separate application. It will be very easy not to use the upload features.

**Comment by jphilip** [comment]
I have been trying to disable uploads including drag/drop and paste. I did not include the ckeditor_uploader app, but I am still getting the same error as above if I omit the upload URL, also an error if I omit the CKEDITOR_UPLOAD_PATH which I don't need.
I was able to hide the upload tabs in the image and link dialogs with 'removeDialogTabs': 'link:upload;image:Upload', in the config. However "removePlugins": ",".join(["upload"]), does not seem to have any effect.
